ingest/src/main/java/com/connexta/ingest/service/api/IngestRequest.java

```diff
@@ -6,9 +6,24 @@
  */
 package com.connexta.ingest.service.api;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.web.multipart.MultipartFile;
+
 /*
  **
- * Java object representation of the json request
+ * Java object representation of the ingest request
  *
  */
-public interface IngestRequest {}
+@Getter
+@Setter
+@AllArgsConstructor
+public class IngestRequest {
```

Comment:
This class can be found in the Ion-Ingest-API in the `InlineObject` class. Should we be using that class instead after the Owasp errors get fixed?

Reply:
I planned on replacing it once we merged your other PR.